agreement/events.go

```diff
@@ -591,10 +596,36 @@ func (e payloadProcessedEvent) ComparableStr() string {
 	return fmt.Sprintf("%v: %.5v", e.t().String(), e.Proposal.BlockDigest.String())
 }
 
+// CredentialTrackingEffect indicates the impact of an event that got filtered
+// on the credential tracking mechanism, for the purpose of tracking the time it
```

Comment:
But this only about late ones, right? If the event was fresh, it would not get marked up with one of these notes.

Reply:
I do like the addition of `Late`. I originally had more objection to "Credential" that anything else, but now I understand why it's called that rather than Proposal.